SERVER/src/client_manager.py

```
@@ -1,52 +1,200 @@
+
+
+CLIENT_HEALTH_CHECK_INTERVAL_SECONDS = 60 * 2
+CLIENT_HEALTH_REQUEST_TIMEOUT_SECONDS = 5
+# Retry settings for send_to_client
+SEND_TO_CLIENT_MAX_RETRIES = 2
+SEND_TO_CLIENT_BASE_DELAY_SECONDS = 1
+SEND_TO_CLIENT_REQUEST_TIMEOUT_SECONDS = 15 # Timeout for the actual /character request
+
+
+class ClientManager:
+    def __init__(self, db: Database):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)